src/exoscale/vinyl/store.clj

```diff
@@ -458,10 +458,10 @@
    ::max-delay           2
    ::initial-delay       2})
 
-(defn continuation-traversing-reducer
-  "A reducer over large ranges.
+(defn continuation-traversing-transducer
```

Comment:
Nitpick: `continuation-traversing-transducer` does an actual transduce, `long-range-reducer` does an actual reduction, Shouldn't they be called `continuation-traversing-transduce` and `long-range-reduce`, similar to `core/transduce` and `core/reduce`?

Reply:
@arnaudgeiser would you be ok with the name change?